salt/cloud/clouds/dimensiondata.py

```diff
@@ -248,7 +248,13 @@ def create(vm_):
         'ex_description': vm_['description'],
         'ex_network_domain': network_domain,
         'ex_vlan': vlan,
-        'ex_is_started': vm_['is_started']
+        'ex_is_started': vm_['is_started'],
+        'ex_ipv4_gateway': vm_['ipv4_gateway'],                     
+        'ex_primary_dns': vm_['primary_dns'],
+        'ex_secondary_dns': vm_['secondary_dns'],
```

Comment:
Are we certain that these values will always be set? I'm just wondering if we should use a `.get()` here instead.

Reply:
So if they aren't mandatory do you agree we should change this so we don't stacktrace? :]

Reply:
Yes, absolutely!

Reply:
Cool. If you'd like to go ahead and make those changes then we should be able to get this in. Thanks!